src/components/Questions/Body.vue

```diff
@@ -212,7 +189,13 @@ export default defineComponent({
 
     function optionBackgroundClass(optionIndex: Number) {
       // returns the background class for the option
```

Comment:
This comment is useless. It's just reading out the function name lol. The function looks like it's doing something more with the `if`. Can you expand the comment a bit?